src/app/zap-templates/zcl/data-model/chip/audio-output-cluster.xml

```diff
@@ -44,7 +44,7 @@ limitations under the License.
   <struct name="AudioOutputInfo">
     <cluster code="0x050b"/>
     <item name="index" type="INT8U"/>
-    <item name="outputType" type="AudioOutputType"/>
+    <item name="outputType" type="AudioOutputType" enum="true"/>
```

Comment:
Why do we need this?  The fact that it's an enum is shoud come from the type, no?

Reply:
zap cannot detect if AudioOutputType is enum or not, we need explicitly to tell this is enum, then zap can query and find the enum fact and tell zpat generation logic that this is enum

Reply:
I would prefer to fix that instead of having us tagging all struct item. All the enums are part of the `ENUM` table so I won't see how ZAP could not detect that `AudioOutputType` is an enum.

Reply:
Exactly.  There is no reason whatsoever for ZAP to not detect that this type is an enum.

Reply:
just try it, if_is_enum meet our requirement, would drop the change for enum=true. Thanks @bzbarsky-apple @vivien-apple